src/asyncapidiff.ts

```diff
@@ -1,44 +1,74 @@
-import { Output, DiffOutputItem } from './types';
+import { Output, OutputType, JSONOutput, Changes } from './types';
 import { breaking, nonBreaking, unclassified } from './constants';
+import convertToYAML from './helpers/output/convertToYAML';
 
 /**
  * Implements methods to deal with diff output.
  * @class
  * @returns {AsyncAPIDiff}
  */
 export default class AsyncAPIDiff {
-  private output: Output;
+  private output: JSONOutput;
+  private outputType: OutputType;
 
-  constructor(output: string) {
+  constructor(output: string, outputType: OutputType) {
```

Comment:
I wonder if we should go with direction to pass as second argument to constructor object of options, not only one option. Do you understand what I mean? For future we can add more options and with shape like:

```js
constructor(output: string, options: { outputType: OutputType }) {
```

we will be able to extend that option object with additional fields without breaking changes. WDYT? :)

Reply:
Oh, yeah. I hadn't thought of that. Love this approach, thanks for the suggestion :+1: